commons/connector/altoida/AltoidaLog.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaLog",
```

Comment:
Maybe we can rename this to `AltoidaAction`, since one record will only describe a single action.